iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergInputFormat.java

```diff
@@ -65,7 +68,7 @@
 
 public class HiveIcebergInputFormat extends MapredIcebergInputFormat<Record>
     implements CombineHiveInputFormat.AvoidSplitCombination, VectorizedInputFormatInterface,
```

Comment:
could we please add `CombineHiveInputFormat.MergeSplit` next to `CombineHiveInputFormat.AvoidSplitCombination`. 
Another thing that i am not sure of why we have both: avoidCombination and merge?

Reply:
Removed `CombineHiveInputFormat.MergeSplit` and added the new method within CombineHiveInputFormat.AvoidSplitCombination.